app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -274,14 +276,14 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
     private fun setupListeners() {
         text_server_name.setOnClickListener {
             ServersBottomSheetFragment().show(
-                activity?.supportFragmentManager,
+                activity!!.supportFragmentManager,
```

Comment:
Avoid `!!`. It was better when it was declared as the way it was before (`activity?.supportFragmentManager`)

Reply:
Same as above, I was getting errors on this line and its a temporary fix.